test/jdk/java/net/InetAddress/IsReachableViaLoopbackTest.java

```diff
@@ -21,22 +21,26 @@
  * questions.
  */
 
-import java.io.*;
-import java.net.*;
-import java.util.*;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+
+import jtreg.SkippedException;
 
 /**
  * @test
  * @bug 8135305
  * @key intermittent
+ * @library /test/lib
  * @summary ensure we can't ping external hosts via loopback if
+ * @run main IsReachableViaLoopbackTest
  */
 
 public class IsReachableViaLoopbackTest {
     public static void main(String[] args) {
         try {
-            InetAddress addr = InetAddress.getByName("localhost");
-            InetAddress remoteAddr = InetAddress.getByName("bugs.openjdk.org");
+            InetAddress addr = InetAddress.getLoopbackAddress();
+            InetAddress remoteAddr = InetAddress.getByAddress("bugs.openjdk.org".getBytes());
```

Comment:
You need to pass a real address here - that is, either byte[4] for an IPv4 address, or byte[16] for an IPv6 address.
For instance - `InetAddress.getByAddress(new byte[] { 104, 77, 118, 65 });` 
With that you should be able to remove the `catch (UnknownHostException)`

Reply:
Thanks, "bugs.openjdk.org" has been replaced as real ipv4 address.